src/foam/mlang/mlang.js

```diff
@@ -564,7 +564,7 @@ foam.CLASS({
           sortedArgs[costKey] = arg;
           costs.push(costKey);
         }
-        costs = costs.sort(foam.Number.compare);
+        costs = costs.sort(foam.util.compare);
```

Comment:
Don't use foam.util.compare if not needed since it will be much slower than specific comparators.